packages/shared/brand.config.json

```diff
@@ -66,7 +66,7 @@
             "availability": "Non-EU"
         },
         "go": {
-            "name": "Deriv Go",
+            "name": "Deriv GO",
```

Comment:
This needs to be fixed in the onboarding page in this file `packages/appstore/src/components/onboarding/static-dashboard.tsx`